ios/RNRadarTests/RNRadarTests.m

```diff
@@ -0,0 +1,188 @@
+//
+//  RNRadarTests.m
+//  RNRadarTests
+//
+//  Created by Russell Cullen on 1/4/19.
+//  Copyright © 2019 Facebook. All rights reserved.
+//
+
+#import <OCMock/OCMock.h>
+#import <XCTest/XCTest.h>
+
+#import "RNRadar.h"
+#import "RNRadarUtils.h"
+
+@interface RNRadarTests : XCTestCase
+
+@end
+
+@interface RNRadar (Test)
+
+- (void)getPermissionsStatusWithResolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject;
+- (void)startTracking:(NSDictionary *)optionsDict;
+- (void)acceptEvent:(NSString *)eventId withVerifiedPlaceId:(NSString *)verifiedPlaceId;
+- (void)rejectEvent:(NSString *)eventId;
+- (void)trackOnceWithResolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject;
+- (void)updateLocation:(NSDictionary *)locationDict resolve:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject;
+
+@end
+
+@implementation RNRadarTests {
+    id radar;
+    id radarUtils;
+    
+    id module;
+}
+
+- (void)setUp {
+    [super setUp];
+    radar = OCMClassMock([Radar class]);
+    OCMStub([radar setDelegate:[OCMArg any]]).andDo(nil);
+    radarUtils = OCMClassMock([RNRadarUtils class]);
+    
+    module = [RNRadar new];
+}
+
+- (void)tearDown {
+    [radarUtils stopMocking];
+    [radar stopMocking];
+    [super tearDown];
+}
+
+- (void)testSetUserId {
+    OCMStub([radar setUserId:[OCMArg any]]).andDo(nil);
+    
+    NSString* userId = @"someUserId123";
```

Comment:
Stylistic thing but `NSString *userId` instead of `NSString* userId`. May want to add linting for native iOS and Android code as well.